airflow/www/app.py

```diff
@@ -151,15 +152,18 @@ def integrate_plugins():
 
                 for v in flask_appbuilder_views:
                     log.debug("Adding view %s", v["name"])
-                    appbuilder.add_view(v["view"],
-                                        v["name"],
-                                        category=v["category"])
+                    if 'view' in v:
+                        v['baseview'] = v.pop('view')
+                        warnings.warn(
+                            "AirflowPlugin.flask_appbuilder_views[] dictionaries should use 'baseview' instead of 'view. "
```

Comment:
Can we include the plugin name in this message to make it easier to track down the plugin at fault (since we can't have the file:line of the warning come from the plugin file.